Examples/Example-iOS/AppDelegate.swift

```diff
@@ -36,9 +36,12 @@ final class AppDelegate: UIResponder, UIApplicationDelegate {
         BacktraceClient.shared?.attributes = ["foo": "bar", "testing": true]
         
         let fileName = "sample.txt"
-        let fileUrl = try? createAndWriteFile(fileName)
+        guard let fileUrl = try? createAndWriteFile(fileName) else {
+            print("Could not create the file attachment")
```

Comment:
```suggestion
            BacktraceLogger.error("Could not create the file attachment")
```

Reply:
I don't think we have access to `BacktraceLogger` here. We use a normal `print` elsewhere in this class too.

Reply:
I'd recommend removing all print statements from the entire codebase and replacing them with proper log levels. This way, we can control how many events are visible for our end users. Please note that the `Could not create the file attachment` log will be added to the application without any other context, so developers might get confused.

Reply:
I can do this but I will have to make BacktraceLogger functions public @krml19

Reply:
@rqbacktrace There is no need for that. Those methods are used within other methods' scopes, so it should work just fine.

Reply:
@krml19 it doesn't build unless I make `BacktraceLogger` functions public. I get `info` is inaccessable due to `internal` protection level.

Reply:
Based on internal discussion with @krml19 we will not make this change

Reply:
It's just an example project. Please ignore my previous comments